### DISCUSSION TITLE
How to use copilot-knowledge-bases?

### DISCUSSION THREAD

**Discussion by skwoojin** [community/community#131361] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

As shown in the picture, you cannot proceed any further on this screen.
I would appreciate it if you could give me an answer on how to configure it.

thank you
![Copilot Knowledge base](https://github.com/community/community/assets/146838814/e6415863-2778-4dc2-b048-b613c3d275c2)

**Comment by mustafaobaidd** [comment]
just copy and paste this :
---
<br/>

**Reply by mustafaobaidd** [reply]
![1](https://github.com/community/community/assets/145908857/e7b2ea87-4ae2-4b33-8dd9-412db7f70944)

**Reply by mustafaobaidd** [reply]
---

**Comment by skwoojin** [comment]
As shown in the picture, you cannot proceed any further on this screen.
I would appreciate it if you could give me an answer on how to configure it.

![Copilot Knowledge base](https://github.com/community/community/assets/146838814/d1deed41-3a83-45c8-92a8-f4ba7ae27230)

**Reply by mustafaobaidd** [reply, marked as answer]
1. **Ensure You Have the Necessary Permissions**: Make sure you have administrative rights in your GitHub organization, as configuring knowledge bases might require higher-level permissions.

2. **Create or Identify Repositories**: Knowledge bases are groups of repositories. Ensure you have the repositories you want to include in your knowledge base.

3. **Access Copilot Settings**:
   - Go to your GitHub organization settings.
   - Navigate to the Copilot settings section.

4. **Configure Knowledge Bases**:
   - In the Copilot settings, there should be an option to configure knowledge bases.
   - Select the repositories you want to include in your knowledge base.

5. **Assign Knowledge Bases to Users**: Members of your organization may need to choose which knowledge base to use. This setting should be available in their individual Copilot settings.

6. **Consult GitHub Documentation**: If you encounter issues or need detailed guidance, refer to the [GitHub Copilot documentation](https://docs.github.com/en/copilot).

**Reply by pr1ntr** [reply]
Does this require an enterprise plan?
